src/main/java/net/dblsaiko/illuminate/client/render/HdrFramebuffer.java

```diff
@@ -0,0 +1,64 @@
+package net.dblsaiko.illuminate.client.render;
+
+import com.mojang.blaze3d.platform.GlConst;
+import com.mojang.blaze3d.platform.GlStateManager;
+import com.mojang.blaze3d.platform.TextureUtil;
+import com.mojang.blaze3d.systems.RenderSystem;
+import net.minecraft.client.gl.SimpleFramebuffer;
+import org.lwjgl.opengl.GL30;
+
+import java.nio.IntBuffer;
+
+public class HdrFramebuffer extends SimpleFramebuffer
+{
+    public HdrFramebuffer(int width, int height, boolean useDepth, boolean getError)
+    {
+        super(width, height, useDepth, getError);
+    }
+
+    public void initFbo(int width, int height, boolean getError)
+    {
+        RenderSystem.assertOnRenderThreadOrInit();
+        int i = RenderSystem.maxSupportedTextureSize();
+        if (width > 0 && width <= i && height > 0 && height <= i)
+        {
+            this.viewportWidth = width;
+            this.viewportHeight = height;
+            this.textureWidth = width;
+            this.textureHeight = height;
+            this.fbo = GlStateManager.glGenFramebuffers();
+            this.colorAttachment = TextureUtil.generateTextureId();
+            if (this.useDepthAttachment)
+            {
+                this.depthAttachment = TextureUtil.generateTextureId();
+                GlStateManager._bindTexture(this.depthAttachment);
+                GlStateManager._texParameter(GlConst.GL_TEXTURE_2D, GlConst.GL_TEXTURE_MIN_FILTER, GlConst.GL_NEAREST);
+                GlStateManager._texParameter(GlConst.GL_TEXTURE_2D, GlConst.GL_TEXTURE_MAG_FILTER, GlConst.GL_NEAREST);
+                GlStateManager._texParameter(GlConst.GL_TEXTURE_2D, GlConst.GL_TEXTURE_COMPARE_MODE, 0);
+                GlStateManager._texParameter(GlConst.GL_TEXTURE_2D, GlConst.GL_TEXTURE_WRAP_S, GlConst.GL_CLAMP_TO_EDGE);
+                GlStateManager._texParameter(GlConst.GL_TEXTURE_2D, GlConst.GL_TEXTURE_WRAP_T, GlConst.GL_CLAMP_TO_EDGE);
+                GlStateManager._texImage2D(GlConst.GL_TEXTURE_2D, 0, GlConst.GL_DEPTH_COMPONENT, this.textureWidth, this.textureHeight, 0, GlConst.GL_DEPTH_COMPONENT, GlConst.GL_FLOAT, (IntBuffer)null);
+            }
+
+            this.setTexFilter(9728);
+            GlStateManager._bindTexture(this.colorAttachment);
+            GlStateManager._texParameter(GlConst.GL_TEXTURE_2D, GlConst.GL_TEXTURE_WRAP_S, GlConst.GL_CLAMP_TO_EDGE);
+            GlStateManager._texParameter(GlConst.GL_TEXTURE_2D, GlConst.GL_TEXTURE_WRAP_T, GlConst.GL_CLAMP_TO_EDGE);
+            GlStateManager._texImage2D(GlConst.GL_TEXTURE_2D, 0, GL30.GL_RGBA16F, this.textureWidth, this.textureHeight, 0, GlConst.GL_RGBA, GlConst.GL_UNSIGNED_BYTE, (IntBuffer)null);
```

Comment:
I assume this is the only line you changed? Try just doing super.initFbo() and this line below instead of copying the entire function

Also that should probably be GL_FLOAT instead of GL_UNSIGNED_BYTE, no?

Reply:
Ah, good idea. Forgot you could modify that parameter after the buffer had been constructed.